### ISSUE TITLE
[filter] Hide form fields that don't have required/aria-required

### ISSUE THREAD

**Issue by erikkroes** [erikkroes/access-tools#9]
By hiding form fields (and labels) that are not required, there is less cognitive load.

**Comment by Malvoz** [comment]
Consider this scenario:

I want to order a pizza, but my favorite pizza includes an ingredient that I'm allergic to so I'll use a non-required `<textarea>` field to write a comment to the restaurant, asking them to exclude the ingredient.

In the above scenario I wont be able to exclude the ingredient because the extension removes the textarea.

**Comment by erikkroes** [comment]
Yes. It would very much be an optional one. Maybe even a concious button press to execute. I'd have to consider.
